docs/gitlab_integration.md

```diff
@@ -0,0 +1,132 @@
+# 1. Install/register GitLab Runner
+GitLab Runner is the open source project that is used to run your jobs and
+send the results back to GitLab. You have to
+[install](https://docs.gitlab.com/runner/install/) it on a machine where
+your analysis will be performed.
+
+Once GitLab Runner is installed, you need to register it with GitLab.
+You can use the following command:
+```sh
+sudo gitlab-runner register \
+    --non-interactive \
+    --url "https://mycompany.gitlab.com" \
+    --registration-token "$REGISTRATION_TOKEN" \
```

Comment:
`$REGISTRATION_TOKEN` means to me that this is an environment variable and I should set token in the REGISTRATION_TOKEN environment variable, which is probably not recommended, because it can be easily read.